.github/dependabot.yml

```diff
@@ -11,3 +11,11 @@ updates:
     schedule:
       # Check for updates to GitHub Actions every week
       interval: "weekly"
+    commit-message:
+      prefix: "BAU"
```

Comment:
I don't think we should use BAU in official docs.  Our policy is that all changes should have a JIRA ticket; so can we change this.